spec/lib/task_helpers/imports/alerts_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe 'TaskHelpers::Imports::Alerts' do
+  let(:data_dir) { File.join(File.expand_path(__dir__), 'data', 'alerts') }
+  let(:alert) { 'Alert_Import_Test.yaml' }
+  let(:bad_alert) { 'Bad_Alert_Import_Test.yml' }
```

Comment:
Can we align these braces? I know it's not a rubocop thing, but it's something we usually do.